src/django/api/views/submission.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.mixins import ListModelMixin
+from rest_framework.permissions import IsAuthenticated
+
+from ..models.submission import Submission
+from ..serializers.submission import SubmissionSerializer
+from ..filters.submission import (
+    UserSubmissionFilterBackend,
+    RequestSubmissionFilterBackend,
+)
+
+
+class SubmissionsViewSet(ListModelMixin, GenericViewSet):
```

Comment:
I've sometimes had trouble making custom actions for ViewSets in the past, so have preferred simple Views, but am willing to give this a shot, in case things have gotten better.

They do document a way to have custom actions: https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

For if we want something like:

```
/api/submissions/42/approve/
```

Reply:
The ViewSet approach might not be the best approach given some of the changes suggested in https://github.com/azavea/iow-boundary-tool/pull/115. The custom actions seem flexible, but I'm guessing not flexible enough.

I think we'll be better off sticking with Simple Views.